Tests/SentryTests/Integrations/Breadcrumbs/SentrySystemEventBreadcrumbsTest.swift

```diff
@@ -324,10 +324,38 @@ class SentrySystemEventBreadcrumbsTest: XCTestCase {
         }
     }
 
+    func testSignificantTimeChangeNotificationBreadcrumb() throws {
+        sut = fixture.getSut(currentDevice: nil)
+
+        fixture.notificationCenterWrapper.post(Notification(name: UIApplication.significantTimeChangeNotification, object: nil))
+
+        XCTAssertEqual(1, fixture.delegate.addCrumbInvocations.count)
+
+        if let crumb = fixture.delegate.addCrumbInvocations.first {
+
+            XCTAssertEqual("device.event", crumb.category)
+            XCTAssertEqual("system", crumb.type)
+            XCTAssertEqual(SentryLevel.info, crumb.level)
+
+            let data = try XCTUnwrap(crumb.data, "no breadcrumb.data")
+            XCTAssertEqual("SIGNIFICANT_TIME_CHANGE", data["action"] as? String)
+        }
```

Comment:
`m`: We can use XCTUnwrap here

```suggestion
        let crumb = try XCTUnwrap(fixture.delegate.addCrumbInvocations.first)

        XCTAssertEqual("device.event", crumb.category)
        XCTAssertEqual("system", crumb.type)
        XCTAssertEqual(SentryLevel.info, crumb.level)

        let data = try XCTUnwrap(crumb.data, "no breadcrumb.data")
        XCTAssertEqual("SIGNIFICANT_TIME_CHANGE", data["action"] as? String)
```